src/OSPSuite.UI/Controls/UxChartControl.cs

```diff
@@ -84,10 +83,11 @@ public virtual void CopyChartToClipboard(ChartControl chartControl)
       {
          using (var ms = new MemoryStream())
          {
-            chartControl.ExportToImage(ms, ImageFormat.Emf);
+            chartControl.ExportToImage(ms, ImageFormat.Png);
             ms.Seek(0, SeekOrigin.Begin);
 
-            _clipboardTask.PutEnhMetafileOnClipboard(this, new Metafile(ms));
```

Comment:
This throws a bunch of excpetion too (GDI stuff etc..). I would get rid off it completely but it's used in the wj export..lol